cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -635,6 +635,7 @@ define(['jquery', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers', 'common/j
                     saveHighlights();
 
                     expectServerHandshakeWithHighlights(updatedHighlights);
+                    expectHighlightLinkNumberToBe(updatedHighlights.length);
```

Comment:
Since the sections that these tests create have no subsections, this check fails without the fix, and passes with the fix.